src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form"
+    data-qa="big"
+    method="post">
+      <input
+      class="form__big"
```

Comment:
suggest better naming which will decrease amount of code in styles.
class="form__field"
it will be common class for big and small input